### PR TITLE
DP 2206 org page schema

### PR DESCRIPTION
**Pull request by jesconstantine**
This PR implements schema.org for the org landing page and provides a hook for adding schema.org data objects to template and page patterns.

Ticket: https://jira.state.ma.us/browse/DP-2206

## To test:

- [ ] Pull down branch
- [ ] Build pattern lab
- [ ] Browse to /?p=pages-ORG-Health-Services
- Inspect the rendered markup, verify:
  - [ ] the `<script type="application/ld+json">` appears just before the `</main>` tag
  - [ ] the properties / values match the expected "Rendered json+ld data object" in the page pattern docs
- [ ] Copy the page pattern markup (inspect the page, grab the pattern `<html>` element from within the patter iframe
- [ ] Verify that you can see the properties picked up in google's [structured data testing tool](https://search.google.com/structured-data/testing-tool/u/0/) (see screenshots below)
- [ ] Verify that meta/schema && page docs are understandable, helpful for potential implementation

### Google SDTT finds governmentOrganization object
image

### Google SDTT finds governmentOrganization properties + connects with "memberOf" global "Commonwealth" Organization
image

### CODE REVIEW

File: styleguide/source/_patterns/04-templates/stacked-row-template.twig [hunk 1]
```diff
@@ -21,5 +21,7 @@
       {% include "@base/placeholder.twig" with {'placeholder':{'text':'Post Content'}} %}
     {% endblock %}
   </div>
+  {% block schemaContent %}
+  {% endblock %}
```
Comment (gender404) [inline styleguide/source/_patterns/04-templates/stacked-row-template.twig:25]: for consistency, can we move this after the `</main>` tag

**Review by gender404 — CHANGES_REQUESTED** [review]
I'll review this on Monday, but I saw this and wanted to make a note.

File: styleguide/source/_patterns/04-templates/stacked-row-template.twig (hunk 1 above)
Comment (jesconstantine) [reply styleguide/source/_patterns/04-templates/stacked-row-template.twig:25]: Yep, good catch.  Thanks.

Comment (jesconstantine) [reply styleguide/source/_patterns/04-templates/stacked-row-template.twig:25]: Addressed in: https://github.com/massgov/mayflower/pull/374/commits/a6b24d0e5e647f1c6227dc147352d3e60181e7a9

I actually moved them all inside of the `<main>` for consistency with how they would be implemented  elsewhere (IE Drupal or Wordpress)

**Review by gender404 — APPROVED** [review]

**Review by gleroux02 — APPROVED** [review]

**Comment by gleroux02** [comment]
@jesconstantine this looks good to me.

**Pull request merged**